src/libraries/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -557,7 +557,7 @@ public unsafe void GetFolderPath_Windows(Environment.SpecialFolder folder)
             Assert.Equal(folderPath, knownFolder);
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotMonoAOT))] 
```

Comment:
this doesn't make sense, why would it be AOT-specific?

Reply:
This needs changed into an `ActiveIssue`. `Array.Resize` either does not work all together under AOT or in the way we're using it.  

https://github.com/dotnet/runtime/blob/c2ec86b1c552ac8a1749f9f98e012f707e325660/src/libraries/Common/src/Interop/Unix/System.Native/Interop.MountPoints.cs#L28

Reply:
https://github.com/dotnet/runtime/issues/60586#issuecomment-1196211359